src/components/TimelineNav/TimelineNav.module.css

```diff
@@ -205,7 +205,7 @@
     color: var(--eds-theme-color-text-neutral-default);
   }
 
-  &:focus-visible {
+  &:focus {
```

Comment:
Did notice from testing this tho is that the read/unread dot indicator does not have a focus indicator (not a result of this pr, is existing in NotificationList in [published storybook](https://chanzuckerberg.github.io/edu-design-system/?path=/story/molecules-lists-notificationlist--default)), filed [ticket](https://app.shortcut.com/czi-edu/story/214162)

Reply:
Damn nice catch 👍